### PR TITLE
upstream variable update

### PR DESCRIPTION
**Pull request by dehann**

### CODE REVIEW

**Comment by dehann** [comment]
intended to replace #493

**Comment by dehann** [comment]
HI @Affie , this becomes `mergeVariableData!` right?

**Comment by Affie** [comment]
I don’t think merge is correct here. Just update. We can implement a graph level wrapper.

**Comment by dehann** [comment]
okay thanks

File: src/SubGraphFunctions.jl [hunk 1]
```diff
@@ -67,21 +67,23 @@ function transferUpdateSubGraph!(dest::AbstractDFG,
...
-      # TODO -- reduce to DFG functions only
-      pp = getKDE(vari)
-      rc2 = size(getPoints(pp))
-      @info "sym=$sym, mem size of val=$rc and $(rc2)"
-      updateFullVertData!(dest, vari, updatePPE=updatePPE)
-    end
   end
+  
+  # pass workload up to DFG -- TODO, should this be updateGraphData! or updateGraphSolverData!
+  DFG.updateGraphSolverData!(src, dest, syms)
```
[the remark this answers is not in the document]
Comment (dehann) [reply src/SubGraphFunctions.jl]: comment obsolete

**Pull request merged**